lib/install/management/appliance.go

```diff
@@ -284,6 +284,32 @@ func (d *Dispatcher) addParaVirtualSCSIController(devices object.VirtualDeviceLi
 	return devices, nil
 }
 
+// addSerialPort is separate from the configLogging because I could not successfully add a serial port after initial VM creation
+// The controller slot kept colliding with the Keyboard no matter whether it was explicitly specified or left as -1 for vSphere to determine
```

Comment:
hmm  govc vm.create -on=false caglar followed by govc device.serial.add -vm caglar seems to be working here